chromium_src/chrome/browser/DEPS

```diff
@@ -6,6 +6,7 @@ include_rules = [
   "+brave/grit",
   "+brave/components/brave_ads",
   "+brave/components/brave_component_updater/browser",
+  "+brave/components/brave_federated",
```

Comment:
this change is needed only if you include something from `brave/components/brave_federated` inside `chromium_src/chrome/browser` files. I don't see any changes in `chromium_src/chrome/browser`.